staging/src/k8s.io/client-go/rest/request_test.go

```diff
@@ -94,9 +93,10 @@ func TestRequestSetsHeaders(t *testing.T) {
 }
 
 func TestRequestWithErrorWontChange(t *testing.T) {
+	gvCopy := v1.SchemeGroupVersion
 	original := Request{
 		err:     errors.New("test"),
-		content: ContentConfig{GroupVersion: &api.Registry.GroupOrDie(api.GroupName).GroupVersion},
+		content: ContentConfig{GroupVersion: &gvCopy},
```

Comment:
It's much better to get these things from a registry than to get them directly from the package. Can it be gotten from a registry living in apimachinery somewhere?